doc/readme.md

```diff
@@ -0,0 +1,335 @@
+
+```java
+    /** Called once at the beginning of the game. */
+    protected void setup() {
+        hero = new MyHero();
+        doSetup = false;
+        controller = new ArrayList<>();
+        setupCameras();
+        painter = new Painter(batch, camera);
+        generator = new RandomWalkGenerator();
+        levelAPI = new LevelAPI(batch, painter, generator, this);
+        initBaseLogger();
+        gameLogger = Logger.getLogger(this.getClass().getName());
+        systems = new SystemController();
+        controller.add(systems);
+        pauseMenu = new PauseMenu<>();
+        controller.add(pauseMenu);
+        levelAPI = new LevelAPI(batch, painter, new WallGenerator(new RandomWalkGenerator()), this);
+        levelAPI.loadLevel(LEVELSIZE);
+        createSystems();
+    }
+```
+
+Wenn Sie nun das Spiel starten, sehen Sie zwar das Level im Dungeon, aber Ihren Helden noch nicht.
+
+### Held zeichnen
+
+Damit der Held im Spiel auch angezeigt werden kann, braucht er ein `AnimationComponent`, denn dann wird das `DrawSystem` den Helden an seiner aktuellen Position zeichnen. Für das `AnimationComponent` benötigen wir zwei `Animation`s. Eine für `idleRight` und eine für `idleLeft`. `Animation`s sind eine Reihe an Bildern, die abwechselnd angezeigt werden. 
+
+Animation können wir mit dem `AnimationBuilder` erzeugen, indem wir `AnimationBuilder.buildAnimation` aufrufen und als Parameter das Verzeichnis mit den Bildern übergeben, die zu einer Animation zusammengefasst werden sollen. Das `AnimationComponent` wird im Konstruktor erstellt.
+
+``` java
+package ecs.entities;
+
+import dslToGame.AnimationBuilder;
+import ecs.components.AnimationComponent;
+import ecs.components.PositionComponent;
+import graphic.Animation;
+import tools.Point;
+
+public class MyHero extends Entity {
+
+    public MyHero() {
+        super();
+        new PositionComponent(this, new Point(0, 0));
+        Animation idleLeft = AnimationBuilder.buildAnimation("character/knight/idleLeft");
+        Animation idleRight = AnimationBuilder.buildAnimation("character/knight/idleRight");
+        new AnimationComponent(this, idleLeft, idleRight);
+    }
+}
+
+```
+Wenn Sie das Spiel jetzt starten, sollten SIe ihren Helden sehen:
+
+![Animation](figs/animation.gif)
+
+### Held bewegen
+
+Damit der Held vom `VelocitySystem` bewegt werden kann, benötigt er das `VelocityComponent`. Das `VelocityComponent` speichert die Maximalgeschwindigkeit des Helden, sowie die aktuelle geschwindigkeit des Helden.
+
+Damit wir den Helden auch per Tastatur steuern können, benötigt er das `PlayableComponent`, denn dann wird das `PlayerSystem` die Werte des Helden auf Knopfdruck aktualisieren.
+
+*Hinweis: Mehr zur Konfiguration der Tastenbelegung erfahren Sie [hier](configuration/readme.md).*
```

Comment:
rewording, siehe @cagix kommentar

Reply:
Bitte außerdem nicht den gesamten Kommentar in Italic, sondern nur das einleitende Wort "Hinweis".